src/main/java/uk/gov/beis/els/util/RendererUtils.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.beis.els.util;
+
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.OutputStream;
+import java.util.List;
+import java.util.function.BiConsumer;
+import javax.imageio.ImageIO;
+import org.apache.batik.transcoder.image.ImageTranscoder;
+import org.jsoup.nodes.Document;
+
+public class RendererUtils {
+
+  /**
+   * Collate a list of SVG documents into a single image, using the provided image writer function.
+   * All images must be of the same dimension. They will be appended vertically.
```

Comment:
Any chance this could put them side-by-side rather than stacked? Would be nice for the two-page fiches because A-series paper maintains its aspect ratio when you put the pages side-by-side - so if the fiche pages are nominally A4, if we put them side-by-side you could print the image on an A4 sheet with each fiche page effectively scaled down to A5

Reply:
Sure, easy to change. I've pushed an update to this PR and updated https://github.com/UKGovernmentBEIS/energy-label-service/pull/156 so you can test it through the screens/API on the other PR